Tutorials/whatiswot/10-Application_Domains/10-Application-Domains.vtt

```diff
@@ -0,0 +1,119 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:06.925 
+In this video, we will give a introduction to the W3C Web of Things including its
+
+00:00:06.925 --> 00:00:13.850 
+application domains and we will continue with architecture patterns, and building blocks in the upcoming videos.
+
+00:00:13.850 --> 00:00:17.125 
+So let's start with what is WoT.
+
+00:00:17.125 --> 00:00:24.200 
+It is a set of standards defined for the interoperability of different Internet
+
+00:00:24.200 --> 00:00:31.275 
+of Things platforms and application domains by the World Wide Web Consortium (W3C).
+
+00:00:31.275 --> 00:00:38.325 
+The WoT aims to preserve and complement existing IoT standards and solutions by providing a
+
+00:00:38.325 --> 00:00:45.375 
+standardized approach for connecting and integrating smart devices, services, and applications into the Web.
+
+00:00:45.375 --> 00:00:56.725 
+Now, let us introduce some of the WoT application domains: Smart Homes, Industrial Applications, Agriculture, and Smart Cities.
```

Comment:
the smart home wiggles but the others don't? Also, there should be a bit more pause after smart cities before moving to next slide

Reply:
smart home was wiggling to show it was being presented next. should i make them all wiggle instead or none of them should wiggle?

Reply:
The wiggling should happen just before the transition in second 59.